docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -47,22 +47,22 @@
     "\n",
     "To run this notebook, you'll need a conda environment with the required dependencies. You can set up a new environment (recommended) and run the jupyter server like:\n",
     "```shell\n",
-    "conda create -n hyp3-mintpy python=3.8 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
+    "conda create -n hyp3-mintpy python=3.10 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
```

Comment:
Update the mintpy restriction here to match what we're coding for, I assume `mintpy>=1.5.2` or `mintpy=1.5.2`